src/mlpack/methods/neighbor_search/ns_model_impl.hpp

```diff
@@ -355,7 +394,10 @@ void NSModel<SortPolicy>::Search(arma::mat&& querySet,
   if (randomBasis)
     querySet = q * querySet;
 
-  Log::Info << "Searching for " << k << " nearest neighbors with ";
+  Log::Info << "Searching for " << k;
+  if (Epsilon() != 0)
+    Log::Info << " approximate (e=" << Epsilon() << ")";
```

Comment:
Mmm do you think so? Maybe some users have problem to visualize unicode characters...